core/fed_avg_optimizer.py

```diff
@@ -147,57 +154,73 @@ def _done_training(self, dmlresult_obj):
 		weights.
 		"""
 		new_weights = dmlresult_obj.results.get('weights')
+		self.job.omega = dmlresult_obj.results.get('omega')
+		self.job.sigma_omega = self.job.omega
 		self._update_weights(new_weights)
 		self.job.job_type = JobTypes.JOB_COMM.name
+		self.job.key = "test"
```

Comment:
Was this intentional? If so, I assume the key won't always be `test`, so how would this normally be set?

Reply:
Key management is coming in next blockchain PR, as has been discussed on #dev.

Reply:
Can you at least add a TODO so it's clear this will be changed?